src/OrchardCore/OrchardCore.ResourceManagement/ResourceManagementConstants.cs

```diff
@@ -0,0 +1,10 @@
+namespace OrchardCore.ResourceManagement;
+
+internal static class ResourceManagementConstants
+{
+    public static readonly char[] DependsOnSeparator = [',', ' '];
+
+    public static readonly char[] CulturesSeparator = [',', ' '];
```

Comment:
These two should be deliberately the same, I think, not to have different conventions. So, just `ParameterValueSeparator`?

Reply:
changed